### ISSUE TITLE
Changing rate ignores column 3, outputs everything on column 3 to column 4

### ISSUE THREAD

**Issue by Bestfast** [Eve-ning/reamberPy#75]
**Describe the bug**
Changing rates to some maps sometime doesn't work and shifts column 3 to 4

**Files used (if any)**

https://osu.ppy.sh/beatmapsets/1595304#mania/3258332 (stella 1.3)

**Comment by Eve-ning** [comment]
For some reason, Stella is saved with a bad X Axis (the first argument), assuming it's converted from somewhere

----
Expected:
`4K: 64, 192, 320, 448`
```
64,0,122,5,0,1:0:0:0:
448,0,159,1,0,1:0:0:0:
192,0,197,1,0,1:0:0:0:
320,0,235,1,0,1:0:0:0:
```
----
Stella
`4K: 64, 192, 321, 449`
```
449,384,1162,5,0,0:0:0:0:
192,384,1213,1,0,0:0:0:0:
321,384,1264,1,0,0:0:0:0:
64,384,1315,128,0,1771:0:0:0:0:
```

**Comment by Eve-ning** [comment]
Fixed, new release 0.1.7
